src/Options/index.js

```diff
@@ -170,7 +170,7 @@ export interface ParseServerOptions {
   sessionLength: ?number;
   /* Max value for limit option on queries, defaults to unlimited */
   maxLimit: ?number;
-  /* Sets wether we should expire the inactive sessions, defaults to true
+  /* Sets wether we should expire the inactive sessions, defaults to true. If false, all new sessions are created with no expiration date.
```

Comment:
Can you fix the spelling error of "whether" here?

Also after you've changed `index.js`, run `npm run definitions` and the other files will be auto generated.

Reply:
Spelling error corrected. Sorry for a dummy question... But for that `npm run definitions` I need to fork the repository and run that command on my local computer?

Reply:
That is ok! There are no dummy questions, we are all here to help.

You have already made a fork of Parse Server 😊 

If you download GitHub Desktop, then visit your fork (https://github.com/lsmilek1/parse-server) and click Code > Open with GitHub Desktop this will help you make the local clone.

From there you can select Repository > Open in Terminal and type `npm run defintions`.

<img width="566" alt="Screen Shot 2021-09-07 at 9 18 22 pm" src="https://user-images.githubusercontent.com/4974769/132336200-7d7c8c86-4d47-4efd-98ad-49055e2a7f25.png">

Then when you return to GitHub desktop, you'll notice that there are changes to the other Options files too. These are for the CLI helper and the docs.

<img width="419" alt="Screen Shot 2021-09-07 at 9 20 44 pm" src="https://user-images.githubusercontent.com/4974769/132336513-2f83b715-8241-419e-8097-e9dd606ed1dc.png">

Then if you select "commit", and then push, the changes will arrive here.

Let me know if you need any help!

Reply:
Oh, thanks for great instructions! There seems to be still something missing in my computer

> Cannot find module '@babel/core'

<img width="586" alt="image" src="https://user-images.githubusercontent.com/41491201/132363268-57591951-c19f-42ff-b06e-39c9ee109455.png">

Reply:
Sorry! I think you'll have to run `npm install` before you run `npm run definitions`

Reply:
That did the trick. Thanks for a great help! Is there anything else I can do?

Reply:
Look good!!